packages/astro/src/core/build/common.ts

```diff
@@ -7,7 +8,11 @@ const STATUS_CODE_PAGES = new Set(['/404', '/500']);
 const FALLBACK_OUT_DIR_NAME = './.astro/';
 
 function getOutRoot(astroConfig: AstroConfig): URL {
-	return new URL('./', astroConfig.outDir);
+	if (astroConfig.output === 'static') {
+		return new URL('./', astroConfig.outDir);
+	} else {
+		return new URL('./', astroConfig.build.client);
```

Comment:
isn't there a getClientRoot() in this file? this function was intended to return the actual root.

Reply:
I don't see a `getClientRoot` function anywhere...